aerich/ddl/__init__.py

```diff
@@ -23,6 +27,8 @@ class BaseDDL:
         'ALTER TABLE "{table_name}" ADD {unique}INDEX "{index_name}" ({column_names})'
     )
     _DROP_INDEX_TEMPLATE = 'ALTER TABLE "{table_name}" DROP INDEX "{index_name}"'
+    _ADD_UNIQUE_TEMPLATE = 'ALTER TABLE "{table_name}" ADD UNIQUE ("{column_name}")'
+    _DROP_UNIQUE_TEMPLATE = 'ALTER TABLE "{table_name}" DROP INDEX IF EXISTS "{column_name}_2"'
```

Comment:
Why drop index says `{column_name}_2` while add index says `{column_name}`?

Looks like a bug here and in mysql, while in sqlite it's ok

Reply:
Separated it to another PR.